templates/zilencer/billing.html

```diff
@@ -11,19 +11,15 @@
 
     {% include 'zerver/billing_nav.html' %}
 
+    <script src="https://checkout.stripe.com/checkout.js"></script>
     {{ render_bundle('translations') }}
-
+    {{ render_bundle('billing') }}
     <div class="portico-landing billing-upgrade-page">
         <div class="hero small-hero"></div>
 
         <div class="page-content">
             <div class="main">
-
-                {% if error_message %}
-                <div class="alert alert-danger" id="error-message-box">
-                    {{ error_message }}
```

Comment:
I feel like we pass error_message from the templates, for other errors?

Reply:
Yeah. We are not currently using this in billing page.